keras_core/utils/dataset_utils.py

```diff
@@ -1,3 +1,8 @@
+import torch 
```

Comment:
This should not be imported here. Torch is not a dependency of the package.

Reply:
Done